wgsl/index.bs

```diff
@@ -1391,6 +1441,8 @@ The size may include non-addressable padding at the end of the type.
 Consequently, loads and stores of a value might access fewer memory locations
 than the value's size.
 
+Note: Recall that a host-shareable [=fixed-size array=] must have its [=element count=] specified as a literal.
```

Comment:
I thought we were going to drop this requirement?

Reply:
I guess the question is whether or not we are planning to back WGSL constants by SPIR-V constants. If the answer is yes, then using them on host-shareable stuff is problematic.

Reply:
> I thought we were going to drop this requirement?

Ah, right.  I missed that edit on this branch.  The note is non-normative: the normative wording is at the definition of host-shareable, there is no further restriction on the element count.

I'll remove the note. Thanks for the close reading!